packages/video_player/video_player_web/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
```suggestion
## 2.1.0
```

This needs to be published to be usable, right? If so, we want to bump the version to `2.1.0`, here and in the `pubspec.yaml` of the package!

Reply:
(Done)

Reply:
I can't see the history since this PR has been force-pushed, but it looks like you undid an existing change.

Reply:
@stuartmorgan Apologies, added back https://github.com/flutter/packages/pull/4551/commits/688077fb194b5803e9180afef1b26c6b71245384

Reply:
The actual package version bump is still reverted, as indicated by CI.